modules/cluster/variables.tf

```diff
@@ -447,3 +447,22 @@ variable "enable_acl" {
   description = "Flag to enable ACL instead of bucket ownership for S3 storage"
   type        = bool
 }
+
+variable "enable_ebscsi_driver" {
+  description = "Flag to enable or disable EBS CSI driver addon"
+  type        = bool
+  default     = true
```

Comment:
I don't think this should be true by default since that would change the behaviour for an existing user. But you should add enable_ebscsi_driver=true to https://github.com/jx3-gitops-repositories/jx3-terraform-eks when this is merged. Probably the version needs to be a variable in jx3-terraform-eks.

Reply:
I can configure it that way. Should I also add _enable_worker_groups_launch_template=true_  [ https://github.com/jx3-gitops-repositories/jx3-terraform-eks](https://github.com/jx3-gitops-repositories/jx3-terraform-eks) when this is merged?